### PR TITLE
Update stable/chronograf/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Delete dulicated word.
Line 71: a Persistent Volume volume

### CODE REVIEW

**Comment by timstoop** [comment]
The reason that it's failing is that some of the requirements of the Chart.yaml have changed recently and this Chart is missing an appVersion key. You could add it and simply set it to the version of the image that's default. That would move this forward and fix it for future commits to this Chart as well.

**Comment by AdamDang** [comment]
@timstoop
Thanks a lot. I've add appVersion key.

File: stable/chronograf/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: chronograf
-version: 0.4.2
+version: 0.4.3
+appVersion: "0.4.3"
```
Comment (sameersbn) [inline stable/chronograf/Chart.yaml]: should probably be 1.3, as seen in https://github.com/AdamDang/charts/blob/48692b2314e44bf3881b5e542e84977bf1101802/stable/chronograf/values.yaml#L5

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/chronograf/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: chronograf
 version: 0.4.3
-appVersion: "0.4.3"
+appVersion: 1.3-alpine
```
Comment (sameersbn) [inline stable/chronograf/Chart.yaml]: you dont need to add the `-alpine` part. it should just be the version part

**Review by sameersbn — CHANGES_REQUESTED** [review]

**Review by sameersbn — APPROVED** [review]

**Pull request merged**